api/api-rules/violation_exceptions.list

```diff
@@ -597,6 +597,7 @@ API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,V
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,VolumeConfiguration,PersistentVolumeRecyclerConfiguration
 API rule violation: names_match,k8s.io/kube-proxy/config/v1alpha1,KubeProxyConfiguration,IPTables
 API rule violation: names_match,k8s.io/kube-scheduler/config/v1,LegacyExtender,EnableHTTPS
+API rule violation: names_match,k8s.io/kube-scheduler/config/v1beta1,KubeSchedulerConfiguration,Parallelism
```

Comment:
why are we doing this?

Reply:
Parallelism was reported as an API violation, so had to run an update on exceptions